compiler/il/ILProps.hpp

```diff
@@ -18,262 +18,4 @@
-      LoadAddress                  = 0x00100000,
-      Max                          = 0x00200000,
-      Min                          = 0x00400000,
-      New                          = 0x00800000, // object allocation opcodes
-      ZeroExtension                = 0x01000000,
-      SignExtension                = 0x02000000,
-      ByteSwap                     = 0x04000000,
-      // Available                 = 0x08000000,
-      // Available                 = 0x10000000,
-      // Available                 = 0x20000000,
-      // Available                 = 0x40000000,
-      // Available                 = 0x80000000,
-      };
-   }
-
-// Property flags for property word 3
-namespace ILProp3
-   {
-   enum
-      {
-      Fence                       = 0x00000001,
-      ExceptionRangeFence         = 0x00000002,
-      LikeUse                     = 0x00000004,
-      LikeDef                     = 0x00000008,
-      SpineCheck                  = 0x00000010,
-      ArrayLength                 = 0x00000020,
-      SignedExponentiation        = 0x00000040, ///< signed/unsigned base with signed exponent
-      UnsignedExponentiation      = 0x00000080, ///< signed/unsigned base with unsigned exponent
-      CompareTrueIfLess           = 0x00000100, ///< Result is true if left <  right.  Set for < <= !=
-      CompareTrueIfGreater        = 0x00000200, ///< Result is true if left >  right.  Set for > >= !=
-      CompareTrueIfEqual          = 0x00000400, ///< Result is true if left == right.  Set for <= >= ==
-      CompareTrueIfUnordered      = 0x00000800, ///< Result is true if left and right are incomparable.  (NaN)
-      CanUseStoreAsAnAccumulator  = 0x00001000,
-      HasBranchChild              = 0x00002000, ///< used in conjunction with jumpwithmultipletargets
-      SkipDynamicLitPoolOnInts    = 0x00004000, ///< do not perform dynamicLiteralPool on any integral constant children of this node (used for lengths of array ops)
-      Abs                         = 0x00008000,
-      VectorReduction             = 0x00010000, ///< Indicates if opcode performs vector reduction that produces scalar result
-      Signum                      = 0x00020000, ///< For Xcmp opcodes
-      ReverseLoadOrStore          = 0x00040000,
-      // Available                = 0x00080000,
-      // Available                = 0x00100000,
-      // Available                = 0x00200000,
-      // Available                = 0x00400000,
-      // Available                = 0x00800000,
-      // Available                = 0x01000000,
-      // Available                = 0x02000000,
-      // Available                = 0x04000000,
-      // Available                = 0x08000000,
-      // Available                = 0x10000000,
-      // Available                = 0x20000000,
-      // Available                = 0x40000000,
-      // Available                = 0x80000000,
-      };
-   }
-
-#endif
+#include "compiler/il/OMRILProps.hpp"
```

Comment:
In extensible classes, we normally do not use "compiler/" at the start of includes (there is one exception but it doesn't apply in this case):

```suggestion
#include "il/OMRILProps.hpp"
```

Is there a reason for needing them here?

Reply:
just an oversight on my part